Common/AR/Ar.md

```diff
@@ -1,97 +1,119 @@
-hidden infrastructure to displaying waypoints for navigation.
-
-The AR toolkit component is comprised of one class: `ArcGISArView`. This is a subclass of QQuickItem, which
-contains the functionality needed to display an AR experience in your application. `ArcGISArView` uses `ARKit` (iOS) or
-`ARCore` (Android) augmented reality framework to display the live camera feed and handle real world tracking
-and synchronization with the Runtime SDK's `SceneView`. The `ArcGISArView` is responsible for starting and
-managing an `ARKit` or `ARCore` session. `ArcGISArView` uses a `LocationDataSource` for getting an initial GPS location
-and when continuous GPS tracking is required.
-
-For details about using `ARKit`, please refer to [Apple's documentation](https://developer.apple.com/augmented-reality).
-For `ARCore`, please refer to [Google's documentation](https://developers.google.com/ar/).
-
-The minimal version of the ArcGIS Runtime SDK for Qt is 100.6.
-
-### Features of the AR component
+Augmented reality experiences are designed to "augment" the physical world with
+virtual content that respects real world scale, position, and orientation of a
+device. In the case of Runtime, a `SceneView` object is used to display 3D
+geographic AR data as virtual content on top of a camera feed of the real,
+physical world.
+
+The AR toolkit component allows quick and easy integration of AR into your
+application for a wide variety of scenarios. The toolkit recognizes the
+following common patterns for AR:
+
+- **Flyover**: Flyover AR lets you explore a scene using your device as a
+window into the virtual world. A typical flyover AR scenario begins with the
+scene’s virtual camera positioned over an area of interest. You can walk around
+and reorient the device to focus on specific content in the scene.
+
+- **Tabletop**: Scene content is anchored to a physical surface, appearing as
+if the scene were a 3D physical model.
+
+- **World-scale**: Scene content is rendered exactly where it would be in the
+physical world as viewed through a device's camera feed. The camera feed
+provides physical context to the data content, with data content rendered on
+top of that feed. This pattern is used in scenarios ranging from viewing hidden
+infrastructure, such as the recorded position of utility lines or pipes below a
+street, to displaying waypoints for navigation.
+
+The AR toolkit component is comprised of one class: `ArcGISArView`. This is a
+subclass of `QQuickItem`, which contains the functionality needed to display an
+AR experience in your application. `ArcGISArView` uses ARKit (iOS) or
+ARCore (Android) augmented reality framework to display the live camera feed
+and handle real world tracking and synchronization with the Runtime SDK's
+`SceneView`. The `ArcGISArView` is responsible for starting and managing an
+ARKit or ARCore session. `ArcGISArView` also uses a `LocationDataSource`
+for getting an initial GPS location when continuous GPS tracking is required.
```

Comment:
"continuous" -> with the "locationTrackingMode" PR, the GPS may be used in initial or continuous tracking. Maybe it's better to remove "continuous"?